src/TinyTrainable.h

```diff
@@ -26,12 +26,14 @@ class TinyTrainable {
       }
     };
 
+  // TODO: why protected instead of private
```

Comment:
if it's private, all the instrument classes that inherit TinyTrainable don't have access to these anymore. if you want to make them private we'd have to make public getter/setter methods.

Reply:
okie i deleted this line and added your comment to the docs/software.md